developer_setup/foreman.md

```diff
@@ -0,0 +1,26 @@
+The ManageIQ application can run as a collection of worker processes, without a centralized server. As a developer it may be easier to start the worker that you are currently working on, rather than starting the whole server and letting it launch the worker for you.
+
+## run_single_worker.rb
+
+The `run_single_worker.rb` script allows a single worker to be started without needing a server to monitor it. You simply provide the worker class and optionally a queue name and port as environment variables and the worker will start.
```

Comment:
We probably want to point out the one major change is the starting ports of the web service and web socket workers.